integration-test/tests/save-prompts.android.spec.js

```diff
@@ -121,7 +127,7 @@ test.describe('Android Save prompts', () => {
             }
 
             await login.submitLoginForm(credentials)
-            await login.promptWasNotShown()
+            await login.shouldNotPromptToSave('android')
```

Comment:
This was testing the wrong thing (the prompt to fill, not the prompt to save). Fixed now.